bigchaindb/commands/bigchain.py

```diff
@@ -146,7 +146,13 @@ def run_init(args):
 def run_drop(args):
     """Drop the database"""
     bigchaindb.config_utils.autoconfigure(filename=args.config, force=True)
-    db.drop(assume_yes=args.yes)
+    try:
+        db_name = 'Unknown database'
+        if db != None: db_name = db.get_database_name()
```

Comment:
I think you could replace the last two lines with this one:
```python
db_name = db.get_database_name() if db else 'Unknown database'
```
and put it before the `try:`